src/sentry/static/sentry/app/views/organizationRepositories.jsx

```diff
@@ -204,7 +204,7 @@ const OrganizationRepositories = React.createClass({
     return {
       loading: true,
       error: false,
-      itemList: null,
+      itemList: [],
```

Comment:
there are some places where we do `loading: !this.state.itemList`, so do those need to be updated to account for an empty array?